css/stylesDesktop.css

```diff
@@ -0,0 +1,305 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter&display=swap");
+
+/* RESIZING FOR DESKTOP VERSION */
+@media only screen and (min-width: 1100px) {
+  /* Disabled Elements */
+
+  /* Enabled Elements */
```

Comment:
- [ ]  [**OPTIONAL**]Good job setting the desktop version of your portfolio, however the value for `min-width` property should be in range of `768px` or wider . Noticed that you did set the `min-width` at `1100px` and wider. Kindly update that before you merge the PR 😄 

![Capture 1](https://user-images.githubusercontent.com/80458199/228853959-cf067fae-6404-47d0-a364-53cff9f93e07.PNG)